Source/WebKit/CMakeLists.txt

```diff
@@ -787,6 +850,38 @@ macro(GENERATE_MESSAGE_SOURCES _output_source _inputs)
 endmacro()
 GENERATE_MESSAGE_SOURCES(WebKit_DERIVED_SOURCES "${WebKit_MESSAGES_IN_FILES}")
 
+# Helper macro which wraps the bindings script
+#   _output_source is a list name which will contain generated sources.(eg. WebKit_SOURCES)
+#   _inputs are .idl files to generate.
+macro(GENERATE_IDL_BINDINGS _output_source _inputs)
+    unset(_input_files)
+    unset(_outputs)
+    foreach (_file IN ITEMS ${_inputs})
+        get_filename_component(_name ${_file} NAME_WE)
+        list(APPEND _input_files ${WEBKIT_DIR}/${_file}.idl)
+        list(APPEND _outputs
+            ${WebKit_DERIVED_SOURCES_DIR}/JS${_name}.mm
+            ${WebKit_DERIVED_SOURCES_DIR}/JS${_name}.h
+        )
+        
+        # FIXME: This only works on macOS
+        # list(APPEND ${_output_source} ${WebKit_DERIVED_SOURCES_DIR}/JS${_name}.mm)
```

Comment:
Is this a problem, or can just be removed?

Reply:
The code is generating Obj-C++ files right now, which don't compile on Linux. It can be removed for now, but will have to be brought back at some point, when I get to porting that to C++